script.js

```diff
@@ -1,12 +1,377 @@
+  chooseEpisodeAndShow.classList.add("chooseLabel")
+
+  let pageNavigation = document.querySelector("nav")
+  pageNavigation.prepend(chooseEpisodeAndShow, selectAShow);
+
+  let showsList = getAllShows();
+
+  let sortedShowsList = showsList.sort(function compare(showA, showB) {
+    if (showA.name.toUpperCase() < showB.name.toLocaleUpperCase()) { return -1 }
+    if (showA.name.toUpperCase() > showB.name.toUpperCase()) { return 1 }
+    return 0;
+  })
+
+  for (let showIndex = 0; showIndex < sortedShowsList.length; showIndex++) {
+    let showOption = document.createElement("option");
+    selectAShow.appendChild(showOption);
+
+    showOption.innerText = showsList[showIndex].name
+    showOption.setAttribute("value", `${showsList[showIndex].id}`)
+  }
+
+  selectAShow.addEventListener("change", selectedShow)
+}
+
+function selectedShow(e) {
+  showID = +e.target.value
+
+  removeCurrentPage()
+  getAllEpisodes(showID)
+}
+
+function removeCurrentPage() {
+  let rootElem = document.getElementById("root");
+  rootElem.innerHTML = "";
+
+  let navigationWrapper = document.querySelector(".navigationWrapper")
+  navigationWrapper.innerHTML = "";
+}
+
+
+//Level 500 part 1 and 2
+
+function sortedShowsList() {
+  let showsList = getAllShows();
+
+  let newShowsList = showsList.sort(function compare(showA, showB) {
+    if (showA.name.toUpperCase() < showB.name.toLocaleUpperCase()) { return -1 }
+    if (showA.name.toUpperCase() > showB.name.toUpperCase()) { return 1 }
+    return 0;
+  })
+
+  return newShowsList
+}
+
+function makePageForShowsList() {
+  let showsList = sortedShowsList()
+
+  let rootElem = document.getElementById("root");
+
+  let ourShowsSection = document.createElement("section");
+  ourShowsSection.classList.add("allShows")
+  rootElem.appendChild(ourShowsSection)
+
+  for (let showIndex = 0; showIndex < showsList.length; showIndex++) {
+    let theShowWrapper = document.createElement("article");
+    theShowWrapper.classList.add("wrapper");                                    //made same class for the articles elements in episodes and shows page, so "addSearchBar()" can be used on both
+    theShowWrapper.classList.add("theShowWrapper")
+
+    let theShowTitle = document.createElement("h3");
+    theShowTitle.classList.add("theShowName");
+    theShowTitle.addEventListener("click", getTheShowEpisodes)
+
+    let theShowContentsWrapper = document.createElement("div");
+    theShowContentsWrapper.classList.add("theShowContentsWrapper");
+
+    let imageContainer = document.createElement("div");
+    imageContainer.classList.add("theShowPictureContainer");
+
+    let theShowImage = document.createElement("img");
+    theShowImage.classList.add("theShowPicture");
+    imageContainer.appendChild(theShowImage);
+
+    let theShowSummary = document.createElement("div");
+    theShowSummary.classList.add("theShowDescription");
+
+    let theShowDetails = document.createElement("article");
```

Comment:
I think it'd make more sense for the details to be in a `<section>` element. It's a bit strange to have an `article` inside an `article` isn't it? 🤔 

https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Using_HTML_sections_and_outlines